libcodechecker/analyze/analyzer.py

```diff
@@ -105,6 +106,10 @@ def __get_skip_handler(args):
         LOG.debug_analyzer('Skip file was not set in the command line')
 
 
+def __mgr_init():
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
```

Comment:
Please add a small documentation about ignoring the SIGINT signal here, so the manager will not stop before the pool.